libcontainer/cgroups/stats.go

```diff
@@ -79,6 +79,8 @@ type BlkioStats struct {
 	IoMergedRecursive       []BlkioStatEntry `json:"io_merged_recursive,omitempty"`
 	IoTimeRecursive         []BlkioStatEntry `json:"io_time_recursive,omitempty"`
 	SectorsRecursive        []BlkioStatEntry `json:"sectors_recursive,omitempty"`
+	ThrottleServiceBytes    []BlkioStatEntry `json:"io_service_bytes,omitempty"`
```

Comment:
I'd prefer the name to be `ThrottleIoServiceBytes` (and I'd expect we have a follow up PR to s/Io/IO for all blkio entry names to keep consistency with other places) and json name to be `throttle_io_service_bytes`, and I think adding these two new fields should be in a separated commit.

Same with `ThrottleServiced`.